react/StoreWrapper.js

```diff
@@ -162,9 +163,9 @@ class StoreWrapper extends Component {
           script={[
             {
               type: 'text/javascript',
-              src: `/pwa/workers/register.js${route.path.match(
+              src: `${rootPath}/pwa/workers/register.js${route.path.match(
                 /\?.*/
-              ) || ''}`,
+              ) || '?'}&scope=${encodeURIComponent(rootPath)}`,
```

Comment:
Should this `'}&` be there?
```suggestion
              ) || '?scope=${encodeURIComponent(rootPath)}`,
```

Reply:
yeah, because I want to always pass the `scope` param instead of only passing when there are no query params

Reply:
Oh, the syntax highlight make me read the code wrong